source/org/zfin/db/postGmakePostloaddb/1148/ZFIN-8809.sql

```diff
@@ -0,0 +1,4 @@
+--liquibase formatted sql
+--changeset rtaylor:ZFIN-8809.sql
+
+select convert_gene_to_type('ZDB-LINCRNAG-050208-65','GENE');
```

Comment:
The name does not describe fully the intention: 
convert gene to type
but you are converting a ncr to a gene. Should it be convert marker to marker or something similar?